CHANGELOG.md

```diff
@@ -2,11 +2,20 @@
 
 ## [1.5](https://github.com/iMicknl/ha-tahoma/tree/1.5) (2020-07-03)
 
```

Comment:
@vlebourl how should I do the current changelog approach?

Reply:
I generate the changelog using https://github.com/github-changelog-generator/github-changelog-generator
But it needs the PR to be merged to integrate the changes in the changelog. 
I propose to keep the changelog out of the PRs in general, and after a successful merge, merge master into branch https://github.com/iMicknl/ha-tahoma/tree/update-changelog and update the changelog.